### PR TITLE
fix issue #14, issue #20

### LINKED ISSUE #14: Add pause functionality
Currently the timer lets you start and stop. There is no way to pause the time. We need to be able to pause the time and then resume from the paused time.

### LINKED ISSUE #20: Color accessibility during 5 to 10 minutes
Hi, I notice that the timer is hard to read with a yellow background and white numbers. It would be nice to change the font color to black or other darker color with the yellow background.
Screenshot from 2020-10-07 15-30-19

### PR DESCRIPTION
**Pull request by yuanLeeMidori**
Hi, I've added the pause and resume feature to the extension. Please let me know if there is anything you want me to modify, including coding style and UI. 

Also, I add one setting of the font color during the rand of 5-10 minutes. I set it to black to improve accessibility.

### CODE REVIEW

**Comment by drgn-hart** [comment]
Thanks for working on this. I will need some time to review and merge as I am preoccupied with some other work. Will try to get back to you by next week

**Comment by yuanLeeMidori** [comment]
No worries! Take your time. Let me know if there is anything I can do

File: src/core/timer.js [hunk 1]
```diff
@@ -22,6 +31,7 @@ chrome.runtime.onMessage.addListener(
  * @param {Number} seconds - seconds for which timer should run
 */
 function startTimer(hours, minutes, seconds) {
+  debugger;
```
Comment (drgn-hart) [inline src/core/timer.js]: Remove debugger statement

**Review by drgn-hart — CHANGES_REQUESTED** [review]
Overall looks good. Can you run _eslint src/**_ and fix the linter errors in the code?

File: src/core/timer.js (hunk 1 above)
Comment (yuanLeeMidori) [reply src/core/timer.js]: Thank you for checking my code. Sorry about leaving the debugger. I've removed it and pushed the change.

**Review by drgn-hart — APPROVED** [review]

**Pull request merged**